src/styles/utils/variable.scss

```
@@ -0,0 +1,16 @@
+$size-calendar-item: 100px;
+$gap-between-item: 1px;
+$padding-calendar-body: 10px;
+$main-color-calendar-item: #eee;
+$hover-color-calendar-item: #ffbfcb;
+$total-item-size: $size-calendar-item + $gap-between-item;
+$width-calendar-body: ($total-item-size * 7 - $gap-between-item);
+
+$days-of-week:
+  mon $total-item-size * 0,
```

```suggestion
  mon 0,
```
we can simplify it here